extension/package.json

```diff
@@ -6,7 +6,8 @@
     "postinstall": "npm link ../signer/web && npm link ../components",
     "start": "npm run devBuild -- -w",
     "devBuild": "webpack --progress --colors",
-    "build": "webpack --progress --colors --bail",
+    "prebuild": "npm run clean",
+    "build": "NODE_ENV=production webpack --progress --colors --bail",
```

Comment:
Yeah, saw that - might be worth considering.

I don't want to do big changes to build system until we have automated full builds on windows.